Controllers/users.js

```diff
@@ -1,10 +1,76 @@
-const User = require("../Models/users");
+import bcrypt from 'bcryptjs';
+import validator from 'validator';
+import jwt from 'jsonwebtoken';
+import User from '../Models/users.js';
+
+const generateJWT = (id) => {
+  const token = jwt.sign(
+    { id },
+    'S2T7iqfnSIL1RWP9N8BCCs5jEgDwYRJ0ZbzNA6XF43dO', // TODO: use process.env.MY_SECRET in production
```

Comment:
You could use the environment variable NODE_ENV to check whether you are in production. Depending on where you deploy your app this variable will probably automatically be set to production (heroku sets the value to production per default)